hail/python/test/hail/linalg/test_linalg.py

```diff
@@ -632,11 +632,10 @@ def test_export_rectangles(self):
                 rect_path = new_local_temp_dir()
                 rect_uri = local_path_uri(rect_path)
 
-                (BlockMatrix.from_numpy(nd, block_size=block_size)
-                    .sparsify_rectangles(rects)
-                    .write(bm_uri, force_row_major=True))
+                bm = BlockMatrix.from_numpy(nd, block_size=block_size)
 
-                BlockMatrix.export_rectangles(bm_uri, rect_uri, rects)
+                bm.sparsify_rectangles(rects).write(bm_uri, force_row_major=True)
```

Comment:
this written BM is never used now, right?

Reply:
you right